.github/actions/templates/avm-validateModuleDeployment/action.yml

```diff
@@ -62,7 +62,9 @@ runs:
     - name: Azure Login
       uses: azure/login@v2
       with:
-        creds: ${{ env.AZURE_CREDENTIALS }}
+        client-id: ${{ env.AZURE_CLIENT_ID }}
```

Comment:
Unrelated changes to this and the `avm.template.module.yml` file

Reply:
Yes, this is for testing purposes only. After all changes are complete, we will revert the changes to these two files. 😃